socialdistribution/api/views.py

```diff
@@ -25,6 +27,11 @@ class AuthorViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin, mixins.L
     pagination_class = pagination.CustomPageNumberPagination
 
     def list(self, request, *args, **kwargs):
+        '''
```

Comment:
Purely a stylistic choice, but I would change these all to triple `"`'s, instead of triple `'`'s